### DISCUSSION TITLE
Is there any relevant documentation to explain the included functions and how to call them, I'm having problems trying to use CRLI for clustering.

### DISCUSSION THREAD

**Discussion by PBemmm** [WenjieDu/PyPOTS#64] [Q&A]
=

**Comment by WenjieDu** [comment]
Hi PB, thank you for raising the discussion here! 👍 
PyPOTS' documentation is still under building. Could you please make the problems you met more specific? So that I can try to help. 😊

**Comment by WenjieDu** [comment]
Hi @PBemmm, PyPOTS docs https://docs.pypots.com/ have been online since this May, and you can find the tutorials of all integrated models in the repo [BrewPOTS ](https://github.com/WenjieDu/BrewPOTS).
